src/util.ts

```diff
@@ -1051,3 +1052,13 @@ function logOrRethrowError(error: any, codesToCatch: number[] = []) {
     throw error;
   }
 }
+
+/**
+ * Converts an Ethers BigNumber to a BN.
+ *
+ * @param bigNumber - An Ethers BigNumber instance.
+ * @returns A BN object.
+ */
+export function ethersBigNumberToBN(bigNumber: BigNumber): BN {
```

Comment:
Is this only used in assets controllers? If so, what do you think about putting this in `assetsUtil` instead of `util`? In the monorepo branch I've moved some things around so that functions within `util` are generic and the more specific functions are moved to their respective packages.

Reply:
Sure thing 👍